app/code/Magento/SalesRule/Test/Mftf/Test/StorefrontCartFixedAmountWholeCartDiscountForBundleProductTest.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontCartFixedAmountWholeCartDiscountForBundleProductTest">
+        <annotations>
+            <features value="SalesRule"/>
+            <stories value="Fixed Amount Cart Price Rule"/>
+            <title value="Checking Fixed Amount Cart Price Rule"/>
+            <description value="Checking Fixed Amount Cart Price Rule"/>
```

Comment:
Can we extend the title and description here by adding a more meaningful information on what does this test do?